pkgs/applications/networking/instant-messengers/signal-desktop/generic.nix

```
@@ -56,6 +56,7 @@
 , version
 , hash
 , url
+, apple-emoji
```

```suggestion
, apple-emoji ? false
```
would make the diff smaller